examples/minimal.yml

```diff
@@ -2,12 +2,12 @@ kernel:
   image: "linuxkit/kernel:4.9.x"
   cmdline: "console=ttyS0 console=tty0 page_poison=1"
 init:
-  - linuxkit/init:42fe8cb1508b3afed39eb89821906e3cc7a70551
+  - linuxkit/init:63eed9ca7a09d2ce4c0c5e7238ac005fa44f564b
   - linuxkit/runc:b0fb122e10dbb7e4e45115177a61a3f8d68c19a9
-  - linuxkit/containerd:60e2486a74c665ba4df57e561729aec20758daed
-onboot:
+  - linuxkit/containerd:18eaf72f3f4f9a9f29ca1951f66df701f873060b
```

Comment:
Updating containerd (and init to the corresponding version) here is intentional. The old version does not start up for some reason:
```
/ # cat /var/log/dhcpcd.log
ctr: rpc error: code = 14 desc = grpc: the connection is unavailable
```